src/Clock.tsx

```diff
@@ -0,0 +1,65 @@
+import { Component } from 'react';
+
+function getRandomName(): string {
+  const value = Date.now().toString().slice(-4);
+
+  return `Clock-${value}`;
+}
+
+type State = {
+  clockName: string,
+  today: Date,
+};
+
+export class Clock extends Component<{}, State> {
+  state = {
+    clockName: 'Clock-0',
+    today: new Date(),
+  };
+
+  timerId = 0;
+
+  componentDidMount() {
+    this.timerId = window.setInterval(() => {
+      this.setState({ today: new Date() });
+      // eslint-disable-next-line
+      console.info(this.state.today.toUTCString().slice(-12, -4));
+    }, 1000);
+
+    window.setInterval(() => {
+      this.setState({ clockName: getRandomName() });
+    }, 3300);
+  }
+  // eslint-disable-next-line
```

Comment:
why do you need eslint disable here?

Reply:
Hm... previously linter did't like empty object in componentDidUpdate. But without that object, I couldn't give previous clockName. I just thought that I can make this task without props, only with state. I hadn't can)) So, now I have added props and fixed it.